src/api/restApi.ts

```diff
@@ -0,0 +1,48 @@
+import wretch from 'wretch';
+
+export class RestApi {
+  protected readonly baseApiUrl = 'google.com';
+
+  protected readonly apiWretch = wretch(this.baseApiUrl);
+
+  protected get = this._get();
```

Comment:
could you plz try to remove these wrappers around methods? in the LykkeExchange repo it was introduced this way because of some specific

i mean try to replace like so

```protected get = (url: string) => this.wretch.url().get()...```